tools/kms-health-check/src/checks.rs

```diff
@@ -134,37 +135,70 @@ pub async fn run_config_validation(config_path: &str) -> Result<HealthCheckResul
                     // threshold = max number of malicious/offline nodes tolerated
                     // For Byzantine fault tolerance, need 2/3 majority for healthy status
                     let threshold = threshold_conf.threshold;
-                    let total_nodes = threshold_conf.peers.len(); // peers list includes self
-                    let min_nodes_required = 2 * (threshold as usize) + 1; // Basic MPC requirement
-                    let min_nodes_for_healthy = (2 * total_nodes).div_ceil(3); // 2/3 majority + 1
+                    let total_nodes = threshold_conf.peers.as_ref().map_or(0, |p| p.len()); // peers list includes self
```

Comment:
I am not sure I get why the total nodes including self is computed this way here, but computed in a different way in `threshold/endpoint.rs`.
It seems to me one of the places are wrong (I think it depends on whether `self` is included in `peers` or not)

Reply:
Yes, thank you. All good now